zanata-war/src/test/java/org/zanata/rest/service/ResourceUtilsTest.java

```diff
@@ -60,13 +68,9 @@ public static void logMemoryForTests() {
     }
 
     @Before
-    public void initializeResourceUtils() {
-        MockitoAnnotations.initMocks(this);
-        resourceUtils =
-                SeamAutowire.instance().reset()
-                    .use("entityManager", mockEm)
-                    .use("localeDAO", mockLocaleDAO)
-                    .autowire(ResourceUtils.class);
+    public void initializeRequestScope() {
+        // NB: this is easier than adding @InRequestScope to all test methods
+        contextController.openRequest();
```

Comment:
You can either do this, or annotate the test methods like the comment says. I think either way is fine, but the annotation might give you more fine-grained control if you want it.